apps/web/pages/availability/index.tsx

```diff
@@ -24,15 +24,15 @@ export function AvailabilityList({ schedules }: RouterOutputs["viewer"]["availab
       const previousValue = utils.viewer.availability.list.getData();
       if (previousValue) {
         const filteredValue = previousValue.schedules.filter(({ id }) => id !== scheduleId);
-        utils.viewer.availability.list.setData({ ...previousValue, schedules: filteredValue });
+        utils.viewer.availability.list.setData(undefined, { ...previousValue, schedules: filteredValue });
```

Comment:
First arg is now the `input` which can be escaped by passing undefined if there isn't one